### PR TITLE
Update libs and fix PHP 8

### PR DESCRIPTION
**Pull request by ozh**
I hope this time won't be the same fucking mess

So far : 
- update Aura SQL to 3.x
- rework all the internal logging system, to match Aura's update
- update all libs (keeping current version constraint)

### CODE REVIEW

**Comment by ozh** [comment]
Builds failing on 7.2 or 7.3 but not on 7.4 don't make fucking sense to me. Any clue anyone? @LeoColomb ?

**Comment by LeoColomb** [comment]
@ozh 
> Your Composer dependencies require a PHP version ">= 7.4.0".

File: includes/vendor/composer/platform_check.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+
+// platform_check.php @generated by Composer
+
+$issues = array();
+
+if (!(PHP_VERSION_ID >= 70400)) {
+    $issues[] = 'Your Composer dependencies require a PHP version ">= 7.4.0". You are running ' . PHP_VERSION  .  '.';
```
Comment (LeoColomb) [inline includes/vendor/composer/platform_check.php:8]: @ozh Here!

File: composer.lock [hunk 2]
```diff
@@ -499,30 +598,34 @@
...
-                "url": "https://api.github.com/repos/spatie/array-to-xml/zipball/15b5229fe8c9c14954b21f2bac0cc003ffbc54a5",
-                "reference": "15b5229fe8c9c14954b21f2bac0cc003ffbc54a5",
+                "url": "https://api.github.com/repos/spatie/array-to-xml/zipball/db39308c5236b69b89cadc3f44f191704814eae2",
+                "reference": "db39308c5236b69b89cadc3f44f191704814eae2",
                 "shasum": ""
             },
             "require": {
                 "ext-dom": "*",
-                "php": "^7.2"
+                "php": "^7.4|^8.0"
```
Comment (LeoColomb) [inline composer.lock:623]: @ozh Plus here, too

**Comment by LeoColomb** [comment]
@ozh PHPUnit upgraded as a bonus 😉 

image

**Review by LeoColomb — APPROVED** [review]
@ozh Enjoy! 🎉 

image

**Comment by ozh** [comment]
Djizeus, you rock. I was completely not searching in the right direction :) Thanks!!

**Pull request merged**